.github/workflows/build_and_test.yml

```diff
@@ -568,3 +568,63 @@ jobs:
           flags: xdp,unittest
           fail_ci_if_error: false
           verbose: true
+
+  test_regressions:
+    name: Run ${{ matrix.engine }}-${{ matrix.sanitizer }} fuzzer for regressions
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: true
+      matrix:
+        sanitizer: [address, undefined, memory]
+        engine: [libfuzzer]
+    container:
+      image: gcr.io/oss-fuzz-base/base-builder
+    steps:
+      - uses: actions/checkout@v4
+      - name: Install prerequisites
+        run: |
+          apt-get update && apt-get install -y cmake autoconf flex bison
+          cd ..
+          cp -r PcapPlusPlus/ $SRC/PcapPlusPlus
+          git clone --depth=1 https://github.com/the-tcpdump-group/libpcap.git $SRC/libpcap
+          cd $SRC/PcapPlusPlus/
+      - name: Compile fuzzer
+        run: |
+          export FUZZING_LANGUAGE=c
+          export ARCHITECTURE=x86_64
```

Comment:
Nitpick but I'm not fan of using environment variable to pass value to a script.

I prefer to properly pass it as parameters, so we can properly check the input, have default and explain what we want.

But not a big deal as it's only for CI and not related to this MR